lib/domodule.js

```diff
@@ -46,4 +109,47 @@ class Domodule {
   find(selector) {
     return this.el.querySelectorAll(selector);
   }
+
+  findOne(selector) {
+    const found = this.find(selector);
+
+    if (found.length) {
+      return found[0];
+    }
+
+    return null;
+  }
+
+  findByName(name) {
```

Comment:
In my test case it's `Domodule.getInstance('ExampleModule').findByName('tester')`